shuttle/db/db_functions.py

```diff
@@ -102,3 +104,56 @@ def username_search(username):
             conn.close()
         # if mybethel can't get the data, then prevent anything from loading
         return abort(503)
+
+
+def commit_schedule_to_db(table, all_locations):
```

Comment:
You could get rid of `_to_db` and have it just be `commit_schedule` since it is in the db_functions.py file so it is inferred that it is to the db.